Undefined name 'valueGetter' in Lib/fontParts/world.py

https://github.com/robotools/fontParts/blob/master/Lib/fontParts/world.py#L405

The plural form __valueGetters__ is defined but not the singular form __valueGetter__.  This will result in a NameError being raised instead of the expected ValueError.  Perhaps in this instance __valueName__ is a better substitute.

[flake8](http://flake8.pycqa.org) testing of https://github.com/robotools/fontParts on Python 3.7.1

$ __flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics__
```
./Lib/fontParts/world.py:405:71: F821 undefined name 'valueGetter'
                    raise ValueError("Unknown sort option: %s" % repr(valueGetter))
                                                                      ^
1     F821 undefined name 'valueGetter'
1
```
__E901,E999,F821,F822,F823__ are the "_showstopper_" [flake8](http://flake8.pycqa.org) issues that can halt the runtime with a SyntaxError, NameError, etc. These 5 are different from most other flake8 issues which are merely "style violations" -- useful for readability but they do not effect runtime safety.
* F821: undefined name `name`
* F822: undefined name `name` in `__all__`
* F823: local variable name referenced before assignment
* E901: SyntaxError or IndentationError
* E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree

--- a/Lib/fontParts/world.py
+++ b/Lib/fontParts/world.py
@@ -402,7 +402,7 @@ class BaseFontList(list):
                 elif hasattr(font.info, valueName):
                     value = getattr(font.info, valueName)
                 else:
-                    raise ValueError("Unknown sort option: %s" % repr(valueGetter))
+                    raise ValueError("Unknown sort option: %s" % repr(valueName))
                 sortable.append(value)
             sortable.append(originalIndex)
             sortable.append(font)

